internal/cmd/egctl/version.go

```diff
@@ -79,6 +79,12 @@ func versions(w io.Writer, containerName, output string) error {
 	}
 
 	for _, pod := range pods.Items {
```

Comment:
also consider looking into line 76, it should be completely fine to run `egctl version` w/o getting server output

Reply:
L76 will fail when `kube-apiserver` is unreachable, what's your concern?

Reply:
it is preferable to show unavailable output in a better way rather than erroring out
here's how `docker` does it
```
docker version
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
Client:
 Cloud integration: v1.0.29
 Version:           20.10.22
 API version:       1.41
 Go version:        go1.18.9
 Git commit:        3a2c30b
 Built:             Thu Dec 15 22:28:41 2022
 OS/Arch:           darwin/amd64
 Context:           default
 Experimental:      true
 ```
 and here's how `kubectl` does it
```
$ kubectl version
WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.  Use --output=yaml|json to get the full version.
Client Version: version.Info{Major:"1", Minor:"25", GitVersion:"v1.25.4", GitCommit:"872a965c6c6526caa949f0c6ac028ef7aff3fb78", GitTreeState:"clean", BuildDate:"2022-11-09T13:36:36Z", GoVersion:"go1.19.3", Compiler:"gc", Platform:"darwin/amd64"}
Kustomize Version: v4.5.7
The connection to the server kubernetes.docker.internal:6443 was refused - did you specify the right host or port?
```

Reply:
sound good, will take a look later.

Reply:
open a ticket for that, https://github.com/envoyproxy/gateway/issues/1054

Reply:
https://github.com/envoyproxy/gateway/pull/1053/commits/a4316a0afd86c8624255620141a0907166433611 maybe help this